taqueria-plugin-taquito/fund.ts

```diff
@@ -0,0 +1,94 @@
+import {
+	getCurrentEnvironment,
+	getCurrentEnvironmentConfig,
+	sendAsyncErr,
+	sendJsonRes,
+	sendWarn,
+} from '@taqueria/node-sdk';
+import { Environment } from '@taqueria/node-sdk/types';
+import { TezosToolkit } from '@taquito/taquito';
+import {
+	configureToolKitWithNetwork,
+	FundOpts as Opts,
+	getDeclaredAccounts,
+	getNetworkInstantiatedAccounts,
+	getNetworkWithChecks,
+} from './common';
+import { performTransferOps, TableRow } from './transfer';
+
+const configureTezosToolKit = (parsedArgs: Opts, env: Environment.t): Promise<TezosToolkit> => {
+	const targetConstraintErrMsg = 'Each environment can only have one target, be it a sandbox or a network';
```

Comment:
Should we segment lines 20 through 24 into its own function, and put in common.ts? It seems like that logic is repeated in a few places.